io-engine/src/lvs/lvs_store.rs

```diff
@@ -391,10 +393,19 @@ impl Lvs {
         name: &str,
         bdev: &str,
         uuid: Option<String>,
+        cluster_size: Option<u32>,
     ) -> Result<Lvs, Error> {
         let pool_name = name.into_cstring();
         let bdev_name = bdev.into_cstring();
-
+        let cluster_size = if let Some(cluster_size) = cluster_size {
```

Comment:
@hrudaya21 as you suggested, please add max limit for cluster size.

Reply:
Kept the max cluster size limit to 1GiB. Based on future usage, We can change the macro value.